test/util.py

```diff
@@ -285,6 +285,7 @@ def create_dummy_batches(
                     np.ones((2, 2, 3)) * float(i + 1) * 25, dtype=np.uint8
                 ),
                 "myvideo.seconds": 0.0,
+                "myvideo.audio": np.empty(0),
```

Comment:
Ever since I added this, a few test cases have been failing. Spent a lot of time, couldn't figure out whats wrong. If you figure out why, do let me know